compass/landice/mesh.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+
+
+def flood_fill(thk):
+    """
+    Remove glaciers and ice-fields that are not connected to the ice sheet.
```

Comment:
I'm trying to avoid getting too semantic about this, but do want this to be a general flood fill?  In which case, you replace `thk` with `field` or something and it can be applied to other situations (but leave a note in the docstring that the typical use is for removing glaciers and ice-fields, etc.  On the hand, I'm not sure if there are any other uses I can imagine in COMPASS.  Also, in the docstring mention that this is on a regular grid and not and mpas grid.  Actually how about calling fhe method `gridded_flood_fill` or something?

And while you're at, you could mention there may be more efficient algorithms.